src/librbd/api/Trash.cc

```diff
@@ -171,6 +170,28 @@ int Trash<I>::move(librados::IoCtx &io_ctx, rbd_trash_image_source_t source,
     return r;
```

Comment:
not exact related, but it looks like we are missing `ictx->state->close()` here.

Reply:
Seems appropriate to close image here, added that